code/game/asteroid.dm

```diff
@@ -1,6 +1,6 @@
 var/global/list/possiblethemes = list("organharvest","cult","wizden","cavein","xenoden","hitech","speakeasy","plantlab", "standroom")
 
-var/global/max_secret_rooms = 10
+var/global/max_secret_rooms = 12
```

Comment:
It shouldn't do anything. Once it's used it's removed from attempting to spawn a room, so after all 9 themes are used no more will spawn.